src/deploy/functions/runtimes/discovery/index.ts

```diff
@@ -60,7 +60,7 @@ export async function detectFromPort(
   port: number,
   project: string,
   runtime: runtimes.Runtime,
-  timeout = 10_000 /* 10s to boot up */
+  timeout = 50_000 /* 10s to boot up */
```

Comment:
Thanks for suggesting the fix. Do you have any insights on 50s as magic number? (vs 30 or 20 or etc)?

Reply:
Nope, just an arbitrary value that works, didn't have time to profile it. The intention was to solely increase the timeout. I wanted to parameterize it but I am not sure how to go about that.